src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/ServiceHearingValuesMapping.java

```diff
@@ -59,13 +63,13 @@ public static ServiceHearingValues mapServiceHearingValues(@Valid SscsCaseData c
                 .screenFlow(null)
                 .vocabulary(null)
                 .caseInterpreterRequiredFlag(HearingChannelUtil.isInterpreterRequired(caseData))
-                .panelRequirements(HearingsPanelMapping.getPanelRequirements(caseData, refData))
+                .panelRequirements(hearingsPanelMapping.getPanelRequirements(caseData, refData))
```

Comment:
given that there is a separate field for judiciary, are the panelRequirements here supposed to include judiciary?